### PR TITLE
Splitted General, SF and India into own categories

### PR DESCRIPTION
**Pull request by gaelletjat**
Splitted General and SF Bay Area into their own categories.
Still have a couple of links in India to review. 
Specifically categories naming

### CODE REVIEW

File: women-in-tech-resources/resources-list.md [hunk 1]
```diff
@@ -33,16 +33,71 @@ If this is your first time looking at the Women in Tech Resources List, spend 30
 - [Moms In Tech - Facebook](https://www.facebook.com/groups/momsintechgroup/)
 
 
+### Meetups and Events for Women & Non-binary Engineers:
+- [Lesbians Who Tech](https://www.eventbrite.com/o/lesbians-who-tech-allies-5082754961)
+- [PyLadies](https://www.pyladies.com/)
+- [Women Who Code - Networks](https://www.womenwhocode.com/networks)
+- [Hacktoberfest](https://hacktoberfest.digitalocean.com/)
```
Comment (alodahl) [inline women-in-tech-resources/resources-list.md]: Could you please move Hacktoberfest to Open Source and Hackathons? People have related events sometimes, but I wouldn’t describe it as a meetup.

File: women-in-tech-resources/resources-list.md [hunk 2]
```diff
@@ -33,16 +33,71 @@ If this is your first time looking at the Women in Tech Resources List, spend 30
...
+- [IEEE WiE](https://wie.ieee.org/events/) - On 10/15/2019, the site looked broken.
+
+
+### Conferences:
+- [Grace Hopper Conference](https://ghc.anitab.org/)
+- [DjangoCon Europe](https://djangocon.eu/)
+- [Write/Speak/Code](https://www.writespeakcode.com/#conference)
+
+
+### Hackatons
```
Comment (alodahl) [inline women-in-tech-resources/resources-list.md]:
```suggestion
### Hackathons
```

File: women-in-tech-resources/resources-list.md (hunk 1 above)
Comment (gaelletjat) [reply women-in-tech-resources/resources-list.md]: Sure

**Comment by gaelletjat** [comment]
The file should be ready for a merge.

**Comment by alodahl** [comment]
Fixes #881

**Pull request merged**